src/Form/matching-config/matching-config-source.js

```diff
@@ -224,7 +224,7 @@ const matchingConfiguration = {
         ddgMatcher: {
             matchers: {
                 unknown: {
-                    match: 'search|filter|subject|title|captcha|mfa|2fa|(two|2).?factor|one-time|otp' +
+                    match: 'search|find|filter|subject|title|captcha|mfa|2fa|(two|2).?factor|one-time|otp|social security number|ssn' +
```

Comment:
Was hoping that the `ssn` stuff would fix a bug but it doesn't. Still good to have it here. The `find` is just a variation on `search` that I had missed.